indispensable/build.gradle.kts

```diff
@@ -197,6 +197,7 @@ kotlin {
 
 exportIosFramework(
     "Indispensable",
+    transitiveExports=false,
```

Comment:
here too

Reply:
removing the need to manually declare all transitive api dependencies as exports. effects multibase and encoding, mostly